mappings/net/minecraft/client/render/entity/DragonFireballEntityRenderer.mapping

```diff
@@ -1,2 +1,11 @@
 CLASS net/minecraft/class_891 net/minecraft/client/render/entity/DragonFireballEntityRenderer
+	FIELD field_21735 FIREBALL Lnet/minecraft/class_1921;
```

Comment:
is this how we generally name render layers? thought it'd be `LAYER`

Reply:
Fixed in 0099bc788.